src/components/preview.tsx

```diff
@@ -182,9 +184,19 @@ export const Preview: Component<Props> = (props) => {
           <button
             type="button"
             class="text-left font-semibold uppercase text-xs md:text-sm px-2 py-3 focus:outline-none -mb-1 md:-mb-0.5 leading-none md:leading-tight"
+            style={{ position: 'relative' }}
             onClick={() => setShowLogs(!showLogs())}
           >
-            Console ({logs().length})
+            <Icon
+              style={{
+                height: '28px',
+                position: 'absolute',
+                top: '7px',
+                left: '2px',
+              }}
```

Comment:
I don't like this. However, the baseline for the chevron gives a really ugly alignment, so I had to position it "manually" to give it a good look.

Reply:
Yeah this because of the fond we are using.. Gordita. This is our brand font, its line height is completely broken... I'm thinking of actually removing it from the playground altogether honestly. Same for Fira Code which the coding font... I thought it'd give a sense of familiarity to people as this is a popular option but I'm not so sure anymore.

I really don't mind these sort of hack, although since we are using windicss (I think), you can do these with classes 🤯 like so `class="h-[26px] absolute top=[7px] left-[2px]"` (I'm not 100% on the syntax).

Reply:
I had written a comment here, but it seems like Github didn't register it...

I totally get that you had to do this. For the context, the reason the alignment is so akward is because of the font we are using for the UI : Gorditta. It appears that its line height is completely off. I had to do similar tricks around the UI and I'm seriously considering dropping it from the playground altogether.

That being said, since we are using [windicss](https://windicss.org/) for the styles, you could probably do it like this :
```suggestion
              class="h-[28px] absolute top-[7px] left-[2px]"
```

Reply:
Thank you for the suggestion. I applied it, and also did the same for the parent (replaced `position: relative` with the class `relative`).